lib/xapi/track.rb

```diff
@@ -91,20 +92,28 @@ def dir
     end
 
     def config
-      @config ||= JSON.parse(File.read(File.join(dir, "config.json")))
+      @config ||= JSON.parse(File.read(config_filename))
     end
 
-    def doc(topic)
-      path = File.join(dir, "docs", topic.upcase)
-      f = Dir.glob("%s.*" % path).first
-      case f
-      when /\.md/
-        File.read(f)
-      when /\.org/
-        Orgmode::Parser.new(File.read(f)).to_markdown
+    def config_filename
+      File.join(dir, "config.json")
+    end
+
+    def document_contents(topic)
+      filename = document_fiename(topic)
+      case filename
+      when /\.md$/
+        File.read(filename)
+      when /\.org$/
+        Orgmode::Parser.new(File.read(filename)).to_markdown
       else
-        ""
+        ''
```

Comment:
> I'll try and be more careful about putting them back how they were.

No don't worry about it, I just wanted to hear your thoughts on it. It's fine here.